app/build.gradle

```diff
@@ -85,17 +85,12 @@ android {
 
     buildTypes {
         buildTypes.each {
-            def serverBase = surveyProps['serverBase'].toString()
             def awsBucket = surveyProps['awsBucket'].toString()
+            def serverBase = "https://${awsBucket}.appspot.com"
             def awsAccessKeyId = surveyProps['awsAccessKeyId'].toString()
             def awsSecretKey = surveyProps['awsSecretKey'].toString()
             def apiKey = surveyProps['apiKey'].toString()
 
-            def signingKey = surveyProps['signingKey'].toString()
```

Comment:
Turns out signing key was not defined in any server config files, decided to remove it